.github/workflows/generate.yml

```diff
@@ -0,0 +1,30 @@
+name: Changelog Generation
+
+on:
+  push:
+  workflow_call:
+
+jobs:
+  generate-changelog:
+    name: Generate changelog
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses:  actions/checkout@v4
+        with:
+          repository: Kong/kong
+          ref: test/changelog-remove-prs-and-jiras
+          path: kong
+
+      - uses: leafo/gh-actions-lua@v10
```

Comment:
I suggest we use an official OpenResty release to run this. The CI should be using same technology developers use on their own machines or we risk not discovering it no longer works on our own machine until too late.

Reply:
Removed. This file is only for test purpose